newrelic/config.go

```diff
@@ -155,4 +155,6 @@ type ProviderConfig struct {
 	NewClient            *nr.NewRelic
 	InsightsInsertClient *insights.InsertClient
 	InsightsQueryClient  *insights.QueryClient
+	AccountID            int
+	PersonalAPIKey       string
```

Comment:
🤔 💭 Potentially use a `bool` value here such as `HasPersonalAPIKey bool` to avoid passing a sensitive value around in the code?

Reply:
I like the sound of this.  We could also take it as a later PR.